config/management/operational/Cargo.toml

```diff
@@ -23,14 +23,20 @@ diem-client = { path = "../../../sdk/client", features = ["blocking"], default-f
 diem-config = { path = "../.."}
 diem-crypto = { path = "../../../crypto/crypto" }
 diem-global-constants = { path = "../../global-constants" }
+diem-infallible = { path = "../../../common/infallible" }
 diem-management = { path = ".." }
 diem-network-address-encryption = {path = "../network-address-encryption"}
 diem-secure-storage = { path = "../../../secure/storage" }
 diem-types = { path = "../../../types" }
 diem-workspace-hack = { path = "../../../common/workspace-hack" }
 diem-temppath = { path = "../../../common/temppath" }
 fallible = { path = "../../../common/fallible" }
+network = { path = "../../../network" }
 transaction-builder = { path = "../../../language/transaction-builder" }
+futures = "0.3.12"
```

Comment:
nit: dependencies on crates that are not our own are generally put at the top (e.g., see anyhow, hex, serde, etc.)